Demo/src/main/java/com/braintreepayments/demo/DemoFeature.java

```diff
@@ -0,0 +1,27 @@
+package com.braintreepayments.demo;
+
+public enum DemoFeature {
+    CREDIT_OR_DEBIT_CARDS(0),
+    PAYPAL(1),
+    VENMO(2),
+    GOOGLE_PAY(3),
+    SAMSUNG_PAY(4),
+    VISA_CHECKOUT(5),
```

Comment:
Should we remove Visa from this demo since we don't support it for v4? (it's also not androidx compatible so crashing without jetifier now)

Reply:
Removing Visa Checkout from demo feature list.